crates/rooch/src/commands/session_key/commands/list.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::cli_types::{CommandAction, WalletContextOptions};
+use async_trait::async_trait;
+use clap::Parser;
+use move_core_types::identifier::Identifier;
+use moveos_types::move_types::FunctionId;
+use moveos_types::moveos_std::object::ObjectID;
+use moveos_types::transaction::FunctionCall;
+use rooch_rpc_api::jsonrpc_types::{
+    AnnotatedFunctionResultView, AnnotatedMoveValueView, StateOptions, StatePageView,
+};
+use rooch_types::address::ParsedAddress;
+use rooch_types::error::{RoochError, RoochResult};
+use std::collections::BTreeMap;
+
+/// List all session keys by address
+#[derive(Debug, Parser)]
+pub struct ListCommand {
+    #[clap(short = 'a', long = "address", value_parser=ParsedAddress::parse, default_value = "default")]
+    /// The account's address to list session keys, if absent, show the default active account.
+    address: ParsedAddress,
+
+    #[clap(flatten)]
+    pub context_options: WalletContextOptions,
+}
+
+#[async_trait]
+impl CommandAction<Vec<BTreeMap<Identifier, AnnotatedMoveValueView>>> for ListCommand {
+    async fn execute(self) -> RoochResult<Vec<BTreeMap<Identifier, AnnotatedMoveValueView>>> {
+        let context = self.context_options.build()?;
+        let mapping = context.address_mapping();
+        let address_addr = self.address.into_account_address(&mapping)?;
+
+        let function_id = "0x3::session_key::handle".parse::<FunctionId>()?;
+        let function_call = FunctionCall::new(function_id, vec![], vec![address_addr.to_vec()]);
+        let client = context.get_client().await?;
+        let view_result = client
+            .rooch
+            .execute_view_function(function_call)
+            .await
+            .map_err(|e| RoochError::ViewFunctionError(e.to_string()))?;
```

Comment:
This function call can be defined in the mod https://github.com/rooch-network/rooch/blob/main/crates/rooch-types/src/framework/session_key.rs

Then call it via client, like

https://github.com/rooch-network/rooch/blob/3197fd97aedc9d1ac76e974d0e029d1c47529f15/crates/rooch/src/commands/session_key/commands/create.rs#L102-L104